src/index.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import { type Element } from "react"
-import ScrollToTop from "./ScrollToTop"
+import ScrollToTop from "./ScrollToTop.jsx"
```

Comment:
Shall we just rename that file to `ScrollToTop.js` and leave this as `import ScrolToTop from "./ScrollToTop"`?

Reply:
It is usually good practice to name files with `jsx` in them `jsx`. I'd rather introduce `rollup-plugin-node-resolve` and add `jsx` as an extension, if we want to remove the `.jsx` from the import.

Reply:
Wasn't that the case till about 2 years ago and now we have all moved on to just `.js` and took it as a community-wide standard?

Reply:
Oh yeah looked into it and you are right :)
I've always stuck with `airbnb`'s styling guide and they still insist on using `jsx` as a file extension, but since then, the React developers did move away from it. Updated the PR to reflect this